org.eclipse.winery.yaml.common/src/test/java/org/eclipse/winery/yaml/common/reader/InvalidSyntaxTests.java

```diff
@@ -0,0 +1,20 @@
+package org.eclipse.winery.yaml.common.reader;
+
+import org.eclipse.winery.yaml.common.exception.MultiException;
+import org.eclipse.winery.yaml.common.reader.yaml.Reader;
+import org.junit.Test;
+
+/**
+ * Intention of this test class is to test the robustness of the parser
+ * Goal is to see no exceptions
+ */
+public class InvalidSyntaxTests {
+    
+    private static String PREFIX = "src/test/resources/builder/invalid_syntax";
+    
+    @Test(expected = MultiException.class)
+    public void missingLineBreakTest() throws MultiException {
```

Comment:
Please rename to `missingLineBreakThrowsException`

Reply:
I think, a comment should be added that no exception should occur. Otherwise, Line 9 does not make sense.